### PR TITLE
Cleanup a variety of sphinx warning and update lib structure.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

**Comment by nsoranzo** [comment]
@jmchilton I was also working on some doc fixes, should I add them to your branch?

**Comment by jmchilton** [comment]
@nsoranzo fine by me

**Comment by nsoranzo** [comment]
Thanks, added. Hopefully those make sense to you too.

**Comment by jmchilton** [comment]
They look great to me! Thanks so much.

File: lib/galaxy/webapps/galaxy/api/histories.py [hunk 1]
```diff
@@ -59,11 +59,14 @@ def __init__(self, app):
...
-        index( trans, deleted='False' )
-        * GET /api/histories:
-            return undeleted histories for the current user
-        * GET /api/histories/deleted:
-            return deleted histories for the current user
+        GET /api/histories
+
+        return undeleted histories for the current user
+
+        GET /api/histories/deleted:
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/histories.py]:
```suggestion
        GET /api/histories/deleted
```

File: lib/galaxy/webapps/galaxy/api/metrics.py [hunk 2]
```diff
@@ -29,19 +29,19 @@ def _deserialize_isoformat_date(self, datestring):
     @expose_api_anonymous
     def create(self, trans, payload, **kwd):
         """
-        create( trans, payload )
-        * POST /api/metrics:
+        POST /api/metrics:
             record any metrics sent and return some status object
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/metrics.py]:
```suggestion
        POST /api/metrics

        Record any metrics sent and return some status object.
```

File: lib/galaxy/webapps/galaxy/api/tool_data.py [hunk 3]
```diff
@@ -20,8 +20,9 @@ class ToolData(BaseAPIController):
     @expose_api
     def index(self, trans, **kwds):
         """
-        GET /api/tool_data: returns a list tool_data tables::
+        GET /api/tool_data
 
+        returns a list tool_data tables
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/tool_data.py]:
```suggestion
        Return a list tool_data tables.
```

File: lib/galaxy/webapps/galaxy/api/tours.py [hunk 4]
```diff
@@ -21,17 +21,18 @@ def __init__(self, app):
...
 
     @expose_api_anonymous_and_sessionless
     def show(self, trans, tour_id, **kwd):
         """
-        load_config( self, trans, Tour_config_file, **kwd )
-        * GET /api/tours/{tour_id}:
-            Read a yaml file containing the specified tour definition
+        GET /api/tours/{tour_id}
+
+        Read a yaml file containing the specified tour definition
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/tours.py]:
```suggestion
        Read a YAML file containing the specified tour definition.
```

File: lib/galaxy/webapps/galaxy/api/webhooks.py [hunk 5]
```diff
@@ -17,7 +17,7 @@ def __init__(self, app):
     @expose_api_anonymous_and_sessionless
     def all_webhooks(self, trans, **kwd):
         """
-        *GET /api/webhooks/
+        GET /api/webhooks/
         Returns all webhooks
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/webhooks.py]:
```suggestion
        GET /api/webhooks/

        Return all webhooks.
```

File: lib/galaxy/webapps/galaxy/api/webhooks.py [hunk 6]
```diff
@@ -28,7 +28,7 @@ def all_webhooks(self, trans, **kwd):
     @expose_api_anonymous_and_sessionless
     def webhook_data(self, trans, webhook_id, **kwd):
         """
-        *GET /api/webhooks/{webhook_id}/data/{params}
+        GET /api/webhooks/{webhook_id}/data/{params}
         Returns the result of executing helper function
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/webhooks.py]:
```suggestion
        GET /api/webhooks/{webhook_id}/data/{params}

        Return the result of executing helper function.
```

File: lib/galaxy/webapps/galaxy/api/workflows.py [hunk 7]
```diff
@@ -544,32 +545,48 @@ def import_new_workflow_deprecated(self, trans, payload, **kwd):
     @expose_api
     def update(self, trans, id, payload, **kwds):
         """
-        * PUT /api/workflows/{id}
-            updates the workflow stored with ``id``
+        PUT /api/workflows/{id}
+        updates the workflow stored with ``id``
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/workflows.py]:
```suggestion
        PUT /api/workflows/{id}

        Update the workflow stored with ``id``.
```

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**